cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/curator-oauth-proxy/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: curator-oauth-proxy
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: curator-auth-proxy
```

Comment:
the role is `auth` proxy while the rolebinding is `oauth`.  Don't know if it matters since its only the names, but wanted to point this out.